src/AWS/Orleans.Persistence.DynamoDB/Provider/DynamoDBGrainStorage.cs

```diff
@@ -326,7 +336,7 @@ internal T ConvertFromStorageFormat<T>(GrainStateRecord entity)
             T dataValue = default;
             try
             {
-                if (binaryData?.Length > 0)
+                if (binaryData is { Length: > 0 })
```

Comment:
I didn't even know this was possible.

Reply:
This method might be invoked frequently. Does this change have a performance impact?